_content/monographs/what-buddhism-is_khin-u-ba.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "What Buddhism Is"
+authors:
+  - "U Ba Khin"
+external_url: "https://ia803205.us.archive.org/13/items/what-buddhism-is_202008/What-Buddhism-Is.pdf"
```

Comment:
Don't use the redirected url 8273839372.us.arch... as these links are not stable.  Just link to the entire entry (/details/whatever) or copy the raw (unredirected) PDF link from the details page (right click, copy link address).